addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -31,5 +35,15 @@ export default {
         });
       });
     }
+  },
+  flushReadQueue() {
+    this.reads = [];
+  },
+  flushWorkQueue() {
+    this.work = [];
+  },
+  flushAll() {
```

Comment:
It'd be great to also add a bit of breathing room between these methods. Adding an extra blank line between each makes it that much easier to read.